src/Service/QueueMpxImportResult.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\media_mpx\Service;
+
+use Lullabot\Mpx\DataService\Media\Media;
+
+/**
+ * Holds the result of trying to queue an mpx item for later import.
+ *
+ * @package Drupal\media_mpx\Service
+ */
+class QueueMpxImportResult {
+
+  /**
+   * Whether the queueing attempt was successful.
+   *
+   * @var bool
+   */
+  private $success;
+
+  /**
+   * The mpx Media item.
+   *
+   * @var \Lullabot\Mpx\DataService\Media\Media
+   */
+  private $mpxMediaItem;
+
+  /**
+   * QueueMpxImportResult constructor.
+   *
+   * @param bool $success
+   *   Whether the queueing attempt was successful or not.
+   * @param \Lullabot\Mpx\DataService\Media\Media $mpx_media
+   *   The mpx Media item.
+   */
+  public function __construct(bool $success, Media $mpx_media) {
```

Comment:
What do you think about flipping this, so the constructor is "subject, status" instead of "status, subject".

Reply:
Done